Description: Always install WPrefs binary to bindir
 Even when setting --with-gnustepdir install the main WPrefs executable
 to bindir (/usr/bin) for FHS compliancy.
Author: Andreas Metzler <ametzler@debian.org>
Origin: vendor
Forwarded: no
Last-Update: 2018-09-02

--- a/configure.ac
+++ b/configure.ac
@@ -857,7 +857,7 @@
     [dnl User specified base path
      wprefs_base_dir="$with_gnustepdir/Applications"
      wprefs_datadir="$wprefs_base_dir/WPrefs.app"
-     wprefs_bindir="$wprefs_base_dir/WPrefs.app"])
+     wprefs_bindir="${bindir}"])
 AC_SUBST([wprefs_datadir])dnl
 AC_SUBST([wprefs_bindir])dnl
 
